src/app_service/service/chat/async_tasks.nim

```diff
@@ -49,9 +53,16 @@ type
 const asyncCheckAllChannelsPermissionsTask: Task = proc(argEncoded: string) {.gcsafe, nimcall.} =
   let arg = decode[AsyncCheckAllChannelsPermissionsTaskArg](argEncoded)
   try:
-    let response = status_communities.checkAllCommunityChannelsPermissions(arg.communityId, arg.addresses)
+    let response = status_communities.checkAllCommunityChannelsPermissionsLight(arg.communityId).result
+    let allChannelsPermissions = response.toCheckAllChannelsPermissionsResponseDto()
+    var finalPermissions = allChannelsPermissions
+    for channelId, channelPermissions in allChannelsPermissions.channels:
+      if  not channelPermissions.viewOnlyPermissions.satisfied and not channelPermissions.viewAndPostPermissions.satisfied:
+        let channelResponse = status_communities.checkCommunityChannelPermissions(arg.communityId, channelId).result
```

Comment:
Actually, I think we might want to just return right away with this.
The "real" check should never be needed when checking **all** channels, because that check is only needed to show the locks and hidden channels.

So my idea would be to return directly here and only when the user goes to the channel, then we check the chain (real check)

Reply:
I will check this.

Reply:
Done, works ok.